HLTrigger/Configuration/python/customizeHLTTrackingForPhaseI2017.py

```diff
@@ -106,130 +106,268 @@ def customizeHLTForPFTrackingPhaseI2017(process):
+	maxLostHits = cms.int32(1),  
+	maxLostHitsFraction = cms.double(0.1),
+	maxNumberOfHits = cms.int32(100),
+	minGoodStripCharge = cms.PSet( refToPSet_ = cms.string( "HLTSiStripClusterChargeCutNone" ) ),
+	minHitsMinPt = cms.int32(3),
+	minNumberOfHitsForLoopers = cms.int32(13),
+	minNumberOfHitsPerLoop = cms.int32(4),
+	minPt = cms.double(0.2),
+	minimumNumberOfHits = cms.int32(4),
+	nSigmaMinPt = cms.double(5.0),
+	pixelSeedExtension = cms.bool(True),
+	seedExtension = cms.int32(1),
+	seedPairPenalty = cms.int32(0),
+	strictSeedExtension = cms.bool(True)
     )
 
-    process.HLTIter1PSetTrajectoryBuilderIT = cms.PSet(
-        inOutTrajectoryFilter = cms.PSet( refToPSet_ = cms.string('HLTIter1PSetTrajectoryFilterInOutIT') ),
-        propagatorAlong = cms.string( "PropagatorWithMaterialParabolicMf" ),
-        trajectoryFilter = cms.PSet(  refToPSet_ = cms.string( "HLTIter1PSetTrajectoryFilterIT" ) ),
-        maxCand = cms.int32( 2 ),
-        ComponentType = cms.string( "CkfTrajectoryBuilder" ),
-        propagatorOpposite = cms.string( "PropagatorWithMaterialParabolicMfOpposite" ),
-        MeasurementTrackerName = cms.string( "hltIter1ESPMeasurementTracker" ),
-        estimator = cms.string( "hltESPChi2ChargeMeasurementEstimator16" ),
-        TTRHBuilder = cms.string( "hltESPTTRHBWithTrackAngle" ),
-        updator = cms.string( "hltESPKFUpdator" ),
-        alwaysUseInvalidHits = cms.bool( False ),
-        intermediateCleaning = cms.bool( True ),
-        lostHitPenalty = cms.double( 30.0 ),
-        useSameTrajFilter = cms.bool(False) # new ! other iteration should have it set to True
+    process.HLTIter1PSetTrajectoryBuilderIT = cms.PSet( 
+	inOutTrajectoryFilter = cms.PSet( refToPSet_ = cms.string('HLTIter1PSetTrajectoryFilterInOutIT') ),
+	propagatorAlong = cms.string( "PropagatorWithMaterialParabolicMf" ),
+	trajectoryFilter = cms.PSet(  refToPSet_ = cms.string( "HLTIter1PSetTrajectoryFilterIT" ) ),
+	maxCand = cms.int32( 2 ),
+	ComponentType = cms.string( "CkfTrajectoryBuilder" ),
+	propagatorOpposite = cms.string( "PropagatorWithMaterialParabolicMfOpposite" ),
+	MeasurementTrackerName = cms.string( "hltIter1ESPMeasurementTracker" ),
+	estimator = cms.string( "hltESPChi2ChargeMeasurementEstimator16" ),
+	TTRHBuilder = cms.string( "hltESPTTRHBWithTrackAngle" ),
+	updator = cms.string( "hltESPKFUpdator" ),
+	alwaysUseInvalidHits = cms.bool( False ),
+	intermediateCleaning = cms.bool( True ),
+	lostHitPenalty = cms.double( 30.0 ),
+	useSameTrajFilter = cms.bool(False) 
     )
+    process.HLTIter1GroupedCkfTrajectoryBuilderIT = cms.PSet(
+	ComponentType = cms.string('GroupedCkfTrajectoryBuilder'),
+	bestHitOnly = cms.bool(True),
+	propagatorAlong = cms.string('PropagatorWithMaterialParabolicMf'),
+	trajectoryFilter = cms.PSet(refToPSet_ = cms.string('HLTIter1PSetTrajectoryFilterIT')),
+	inOutTrajectoryFilter = cms.PSet(refToPSet_ = cms.string('HLTIter1PSetTrajectoryFilterInOutIT')),
+	useSameTrajFilter = cms.bool(False),
```

Comment:
Are the two trajectory filters now exactly the same (I didn't check)? If yes, it would be more clear to set `useSameTrajFilter=True` and use same value for `inOutTrajectoryFilter` as for `trajectoryFilter`. If no, why?